src/uct/ib/rc/base/rc_iface.h

```diff
@@ -300,6 +308,8 @@ struct uct_rc_iface {
         uct_rc_send_handler_t  atomic64_ext_handler;  /* 64bit extended */
     } config;
 
+    uint16_t                 flags;
```

Comment:
move it under config as
`uint8_t flush_remote;`
fc_enabled, ooo_rw, tx_poll_always are also "boolean"

Reply:
> move it under config as
> `uint8_t flush_remote;`

done